model/task_lifecycle_test.go

```diff
@@ -2228,6 +2228,20 @@ func TestTryResetTask(t *testing.T) {
 			So(v.Insert(), ShouldBeNil)
 			So(anotherTask.Insert(), ShouldBeNil)
 
+			commitQueueMax := evergreen.GetEnvironment().Settings().CommitQueue.MaxSystemFailedTaskRetries
```

Comment:
Do we set this MaxSystemFailedTaskRetries to something for this test? I'm a little confused about how this reset doesn't work, since I'd assume that this number is just 0 but I think i'm missing something.